scss/style.scss

```diff
@@ -1,23 +1,138 @@
+body {
+    font-family: 'Open Sans', sans-serif;
+    background-color: #f0ecec;
+    padding-top: 50px;
+}
+
+.calculator {
+    
+    width: 330px;
+    height: 430px;
+    background-color: #797979;
+    margin: 0 auto;
+    border-radius: 10px 10px 5px 5px;
+    border: 3px solid rgb(30, 4, 77);
```

Comment:
I see that you use `rgb(30, 4, 77)` color four times. I would suggest extracting it to separate variable (e.g. `$black`) and use it where needed. See SASS docs: https://sass-lang.com/documentation/variables.